device/accton/x86_64-accton_as4630_54pe-r0/plugins/eeprom.py

```diff
@@ -11,7 +8,7 @@
     from sonic_eeprom import eeprom_base
     from sonic_eeprom import eeprom_tlvinfo
     import subprocess
-except ImportError, e:
+except ImportError as e:
```

Comment:
is this a must? I am wondering if the vendor's plugin will have other python3 incompatible issues, it seems you only find one of it.

Reply:
It is a must to make decode-syseeprom work, now that it is Python 3. Technically, with the addition of syseepromd, we should be able to replace decode-syseeprom with simple reads from the DB. But until then, we should maintain backward compatibility with the existing CLI utility while it is still being used.

I found and fixed a few common Python 3 issues with the eeprom.py plugin. It's possible there are Python 3 compatibility issues with other plugins, but these plugins are deprecated in favor of the new platform API. I do not intend to maintain support for the old platform plugins.

Rather than changing these plugins, we could remove decode-syseeprom altogether, but I felt that was a more disruptive change which is best suited for a separate PR.